diff -urN lprof-1.09.orig/ICCtoIT8/Makefile lprof-1.09/ICCtoIT8/Makefile
--- lprof-1.09.orig/ICCtoIT8/Makefile	2003-12-08 18:25:03.000000000 -0500
+++ lprof-1.09/ICCtoIT8/Makefile	2003-12-08 18:27:21.000000000 -0500
@@ -1,6 +1,6 @@
 SHELL = /bin/sh
 
-CFLAGS = -g -O4 -fomit-frame-pointer -Wall -I../include -I/usr/include -I/usr/local/include
+CFLAGS = -g -O4 -fomit-frame-pointer -Wall -I../include -I../../lcms-1.09/include -I/usr/include -I/usr/local/include
 BASEDIR = $(DESTDIR)/usr
 BINDIR  = ..
 
diff -urN lprof-1.09.orig/Makefile lprof-1.09/Makefile
--- lprof-1.09.orig/Makefile	2003-12-08 18:25:03.000000000 -0500
+++ lprof-1.09/Makefile	2003-12-08 18:27:21.000000000 -0500
@@ -1,17 +1,17 @@
 SHELL = /bin/sh
 
 init: FORCE
-	@$(MAKE) -s all
+	@$(MAKE) all
 
 all install clean test:: 
 	@echo "little cms profiler construction set 1.09 being made $@."
 	@echo "please wait ..."
-	cd src ; $(MAKE) $@ ; cd .. 
-	cd ICCtoIT8 ; $(MAKE) $@ ; cd .. 
-	cd qt/qtMeasurementTool; $(MAKE) $@ ; cd ../.. 	
-	cd qt/qtMonitorProfiler; $(MAKE) $@ ; cd ../.. 	
-	cd qt/qtScannerProfiler; $(MAKE) $@ ; cd ../.. 	
-	cd qt/qtProfileChecker;  $(MAKE) $@ ; cd ../.. 	
+	( cd src ; $(MAKE) )
+	( cd ICCtoIT8 ; $(MAKE) )
+	( cd qt/qtMeasurementTool; $(MAKE) $@ )
+	( cd qt/qtMonitorProfiler; $(MAKE) $@ )
+	( cd qt/qtScannerProfiler; $(MAKE) $@ )
+	( cd qt/qtProfileChecker;  $(MAKE) $@ )
 
 FORCE:
 
diff -urN lprof-1.09.orig/qt/qtMeasurementTool/makefile lprof-1.09/qt/qtMeasurementTool/makefile
--- lprof-1.09.orig/qt/qtMeasurementTool/makefile	2003-12-08 18:25:03.000000000 -0500
+++ lprof-1.09/qt/qtMeasurementTool/makefile	2003-12-08 18:27:21.000000000 -0500
@@ -17,8 +17,8 @@
 endif
 
 # some settings
-INCLUDES = -I../../include -I$(QTDIR)/include -I. -I.. -I$(QTDIR)/include/qt $(KDEINCLUDE)
-LIBS     = -L$(QTDIR)/lib -L../../src $(KDELIBDIR)
+INCLUDES = -I../../include -I../../../lcms-1.09/include -I$(QTDIR)/include -I. -I.. -I$(QTDIR)/include/qt $(KDEINCLUDE)
+LIBS     = -L$(QTDIR)/lib -L../../src $(KDELIBDIR) -L../../../lcms-1.09/src
 CFLAGS   = -O4 -Wall $(KDEFLAGS)
 POSTLIB  = $(KDEPOSTLIB) -lqt -llprof -llcms -lm
 
diff -urN lprof-1.09.orig/qt/qtMeasurementTool/qtmeasurementtooldialog.cpp lprof-1.09/qt/qtMeasurementTool/qtmeasurementtooldialog.cpp
--- lprof-1.09.orig/qt/qtMeasurementTool/qtmeasurementtooldialog.cpp	2003-12-08 18:25:03.000000000 -0500
+++ lprof-1.09/qt/qtMeasurementTool/qtmeasurementtooldialog.cpp	2003-12-08 18:27:21.000000000 -0500
@@ -39,6 +39,7 @@
 #include "qtlcmswidgets.h"
 #include "qtmeasurementtooldialog.h"
 
+
 #ifdef USE_KDE
 #include <kfiledialog.h>
 #endif
@@ -402,6 +403,7 @@
 {
   QCanvasView::resizeEvent( e );
   canvas()->resize(width()-4,height()-4);
+  MeasurementDialog.Reload();
 }
 
 void FigureEditor::contentsMousePressEvent(QMouseEvent* e)
@@ -429,14 +431,16 @@
 
     }
     else
+#if 0
     if (e -> button() & Qt::RightButton) {
-        
+#else
+    if (e -> state() & Qt::ShiftButton) {
+#endif
             sizing = (GridItem*) *it;                   
             contentsMouseMoveEvent(e);
     }
     else 
     if (e -> button() & Qt::LeftButton) {
-
             moving = (GridItem*) *it;
             moving_start = e->pos();
     }
@@ -450,6 +454,7 @@
 
 void FigureEditor::contentsMouseMoveEvent(QMouseEvent* e)
 {
+
     if ( moving ) {
 
         moving->moveBy(e->pos().x() - moving_start.x(),
@@ -529,7 +534,7 @@
 
     Canvas = new QCanvas(w-4, h-4); 
     Canvas -> setAdvancePeriod(30);   
-    FigEditor = new FigureEditor(*Canvas, CanvasFrame);
+    FigEditor = new FigureEditor(*this, *Canvas, CanvasFrame);
     FigEditor -> clear();
     //  FigEditor-> resize(w, h);
     CanvasFrameLayout->addWidget(FigEditor);
@@ -839,6 +844,14 @@
 
 }
 
+void qtMeasurementToolDialog::Reload()
+{
+    TheImage -> Reload(fn, MonitorProfileSelector ->getFilename(),
+                                 InputProfileSelector->getFilename(),  
+                                 Canvas);
+}
+
+
 
 // ------------------------------------------------------------------------------ Slots
 
@@ -910,7 +923,7 @@
 
             } else
             {
-                Status->setText("Click & drag mouse to place the grid: Left - Move, Right - Size, Ctrl - UnShear. PICK when finished.");
+                Status->setText("Click & drag mouse to place the grid: Left - Move, Shift - Size, Ctrl - UnShear. PICK when finished.");
                 PickBtn->show();                                        
                 ProofToggle -> show();
             }
diff -urN lprof-1.09.orig/qt/qtMeasurementTool/qtmeasurementtooldialog.h lprof-1.09/qt/qtMeasurementTool/qtmeasurementtooldialog.h
--- lprof-1.09.orig/qt/qtMeasurementTool/qtmeasurementtooldialog.h	2003-12-08 18:25:03.000000000 -0500
+++ lprof-1.09/qt/qtMeasurementTool/qtmeasurementtooldialog.h	2003-12-08 18:27:21.000000000 -0500
@@ -148,6 +148,7 @@
 };
 
 
+class qtMeasurementToolDialog;
 
 // This class implements a visual dragger for pick templates
 
@@ -157,7 +158,7 @@
 
 public:
 
-	FigureEditor(QCanvas& c, QWidget* parent=0, const char* name=0, WFlags f=0) : QCanvasView(&c,parent,name,f) {}    
+	 FigureEditor(qtMeasurementToolDialog &measurement_dialog, QCanvas& c, QWidget* parent=0, const char* name=0, WFlags f=0) : QCanvasView(&c,parent,name,f), MeasurementDialog(measurement_dialog) {}
     void clear();
     void resizeEvent( QResizeEvent* e );
 protected:
@@ -174,7 +175,7 @@
 	GridItem* sizing;
 	GridItem* unshearing;
     QPoint moving_start;
-	
+    qtMeasurementToolDialog &MeasurementDialog;	
 };
 
 
@@ -201,8 +202,7 @@
 
 	BOOL LoadConfig();
 	void SaveConfig();
-
-
+	void Reload();
 
 public slots:
 
diff -urN lprof-1.09.orig/qt/qtMeasurementTool/qtmeasurementtooldialogbase.ui lprof-1.09/qt/qtMeasurementTool/qtmeasurementtooldialogbase.ui
--- lprof-1.09.orig/qt/qtMeasurementTool/qtmeasurementtooldialogbase.ui	2003-12-08 18:25:03.000000000 -0500
+++ lprof-1.09/qt/qtMeasurementTool/qtmeasurementtooldialogbase.ui	2003-12-08 18:27:21.000000000 -0500
@@ -1674,7 +1674,7 @@
                                         </property>
                                         <property>
                                             <name>FileMask</name>
-                                            <string>*.itx</string>
+                                            <string>*.itx;*.ITX</string>
                                         </property>
                                         <property>
                                             <name>Prompt</name>
@@ -1861,7 +1861,7 @@
                                         </property>
                                         <property>
                                             <name>FileMask</name>
-                                            <string>*.icc; *.icm</string>
+                                            <string>*.icc; *.icm; *.ICC; *.ICM</string>
                                         </property>
                                         <property>
                                             <name>Prompt</name>
@@ -1925,7 +1925,7 @@
                                         </property>
                                         <property>
                                             <name>FileMask</name>
-                                            <string>*.icc;*.icm</string>
+                                            <string>*.icc;*.icm;*.ICC;*.ICM</string>
                                         </property>
                                         <property>
                                             <name>Prompt</name>
diff -urN lprof-1.09.orig/qt/qtMonitorProfiler/Makefile lprof-1.09/qt/qtMonitorProfiler/Makefile
--- lprof-1.09.orig/qt/qtMonitorProfiler/Makefile	2003-12-08 18:25:03.000000000 -0500
+++ lprof-1.09/qt/qtMonitorProfiler/Makefile	2003-12-08 18:27:21.000000000 -0500
@@ -18,8 +18,8 @@
 
 
 # some settings
-INCLUDES = -I../../include -I$(QTDIR)/include -I. -I.. -I$(QTDIR)/include/qt  $(KDEINCLUDE)
-LIBS     = -L$(QTDIR)/lib -L../../src $(KDELIBDIR)
+INCLUDES = -I../../include -I../../../lcms-1.09/include -I$(QTDIR)/include -I. -I.. -I$(QTDIR)/include/qt  $(KDEINCLUDE)
+LIBS     = -L$(QTDIR)/lib -L../../src $(KDELIBDIR) -L../../../lcms-1.09/src
 CFLAGS   = -O4 -Wall $(KDEFLAGS)
 POSTLIB  = $(KDEPOSTLIB) -lqt -llprof -llcms -lm
 BINDIR   = ../../
diff -urN lprof-1.09.orig/qt/qtMonitorProfiler/qtmonitorprofilerdialogbase.ui lprof-1.09/qt/qtMonitorProfiler/qtmonitorprofilerdialogbase.ui
--- lprof-1.09.orig/qt/qtMonitorProfiler/qtmonitorprofilerdialogbase.ui	2003-12-08 18:25:03.000000000 -0500
+++ lprof-1.09/qt/qtMonitorProfiler/qtmonitorprofilerdialogbase.ui	2003-12-08 18:27:21.000000000 -0500
@@ -1261,7 +1261,7 @@
                                 </property>
                                 <property>
                                     <name>FileMask</name>
-                                    <string>IT8/CGATS.5 files (*.it? *.cgs);;All files (*.*)</string>
+                                    <string>IT8/CGATS.5 files (*.it? *.cgs *.IT? *.CGS);;All files (*.*)</string>
                                 </property>
                                 <property>
                                     <name>Prompt</name>
@@ -1353,7 +1353,7 @@
                                 </property>
                                 <property>
                                     <name>FileMask</name>
-                                    <string>icc profiles (*.icc *.icm);;All files (*.*)</string>
+                                    <string>icc profiles (*.icc *.icm *.ICC *.ICM);;All files (*.*)</string>
                                 </property>
                                 <property>
                                     <name>Prompt</name>
diff -urN lprof-1.09.orig/qt/qtProfileChecker/makefile lprof-1.09/qt/qtProfileChecker/makefile
--- lprof-1.09.orig/qt/qtProfileChecker/makefile	2003-12-08 18:25:03.000000000 -0500
+++ lprof-1.09/qt/qtProfileChecker/makefile	2003-12-08 18:27:21.000000000 -0500
@@ -18,8 +18,9 @@
 
 
 # some settings
-INCLUDES = -I../../include -I$(QTDIR)/include -I. -I.. -I$(QTDIR)/include/qt  $(KDEINCLUDE)
-LIBS     = -L$(QTDIR)/lib -L../../src $(KDELIBDIR)
+INCLUDES = -I../../include -I../../../lcms-1.09/include -I$(QTDIR)/include -I. -I.. -I$(QTDIR)/include/qt  $(KDEINCLUDE)
+LIBS     = -L$(QTDIR)/lib -L../../src $(KDELIBDIR) -L../../../lcms-1.09/src
+
 CFLAGS   = -O4 -Wall $(KDEFLAGS)
 POSTLIB  = $(KDEPOSTLIB) -lqt -llprof -llcms -lm
 BINDIR   = ../../
diff -urN lprof-1.09.orig/qt/qtProfileChecker/qtprofilecheckerdialogbase.ui lprof-1.09/qt/qtProfileChecker/qtprofilecheckerdialogbase.ui
--- lprof-1.09.orig/qt/qtProfileChecker/qtprofilecheckerdialogbase.ui	2003-12-08 18:25:03.000000000 -0500
+++ lprof-1.09/qt/qtProfileChecker/qtprofilecheckerdialogbase.ui	2003-12-08 18:27:21.000000000 -0500
@@ -1432,7 +1432,7 @@
                                 </property>
                                 <property>
                                     <name>FileMask</name>
-                                    <string>*.icm;*.icc;*.pf</string>
+                                    <string>*.icm;*.icc;*.pf;*.ICM;*.ICC;*.PF</string>
                                 </property>
                                 <property>
                                     <name>Prompt</name>
diff -urN lprof-1.09.orig/qt/qtScannerProfiler/Makefile lprof-1.09/qt/qtScannerProfiler/Makefile
--- lprof-1.09.orig/qt/qtScannerProfiler/Makefile	2003-12-08 18:25:03.000000000 -0500
+++ lprof-1.09/qt/qtScannerProfiler/Makefile	2003-12-08 18:27:21.000000000 -0500
@@ -18,8 +18,8 @@
 
 
 # some settings
-INCLUDES = -I../../include -I$(QTDIR)/include -I. -I.. -I$(QTDIR)/include/qt  $(KDEINCLUDE)
-LIBS     = -L$(QTDIR)/lib -L../../src $(KDELIBDIR)
+INCLUDES = -I../../include -I../../../lcms-1.09/include -I$(QTDIR)/include -I. -I.. -I$(QTDIR)/include/qt  $(KDEINCLUDE)
+LIBS     = -L$(QTDIR)/lib -L../../src $(KDELIBDIR) -L../../../lcms-1.09/src
 CFLAGS   = -O4 -Wall $(KDEFLAGS)
 POSTLIB  = $(KDEPOSTLIB) -lqt -llprof -llcms -lm
 BINDIR   = ../../
diff -urN lprof-1.09.orig/src/makefile lprof-1.09/src/makefile
--- lprof-1.09.orig/src/makefile	2003-12-08 18:25:03.000000000 -0500
+++ lprof-1.09/src/makefile	2003-12-08 18:27:21.000000000 -0500
@@ -1,6 +1,6 @@
 SHELL = /bin/sh
 
-CFLAGS  = -g -O4 -I../include -fomit-frame-pointer -Wall 
+CFLAGS  = -g -O4 -I../include -I../../lcms-1.09/include -fomit-frame-pointer -Wall 
 BASEDIR = $(DESTDIR)/usr
 BINDIR  = $(BASEDIR)/bin
 LIBDIR  = $(BASEDIR)/lib
