diff -rupN iguanaIR-1.0.1/iguanaIR.init iguanaIR-1.0.1.new/iguanaIR.init
--- iguanaIR-1.0.1/iguanaIR.init	2010-11-21 12:28:28.000000000 -0500
+++ iguanaIR-1.0.1.new/iguanaIR.init	2010-11-28 21:11:19.000000000 -0500
@@ -20,7 +20,7 @@
 ######################
 
 # load the default settings
-[ -f /etc/default/iguanaIR ] && . /etc/default/iguanaIR
+[ -f /etc/iguanaIR/iguanaIR ] && . /etc/iguanaIR/iguanaIR
 # combine some variables into a command line
 IGUANAIR_OPTIONS="$OPTIONS --log-level=$LOGLEVEL --send-timeout=$SENDTIMEOUT --receive-timeout=$RECEIVETIMEOUT $DRIVERS"
 
diff -rupN iguanaIR-1.0.1/iguanaIR.options iguanaIR-1.0.1.new/iguanaIR.options
--- iguanaIR-1.0.1/iguanaIR.options	2009-11-16 21:26:47.000000000 -0500
+++ iguanaIR-1.0.1.new/iguanaIR.options	2010-11-28 21:27:46.000000000 -0500
@@ -58,4 +58,4 @@ SENDTIMEOUT=1000
 #DISTRO=fedora
 #DISTRO=debian
 #DISTRO=ubuntu
-#DISTRO=gentoo
+DISTRO=gentoo 
diff -rupN iguanaIR-1.0.1/Makefile.in iguanaIR-1.0.1.new/Makefile.in
--- iguanaIR-1.0.1/Makefile.in	2010-11-20 19:28:34.000000000 -0500
+++ iguanaIR-1.0.1.new/Makefile.in	2010-11-28 21:07:08.000000000 -0500
@@ -115,17 +115,17 @@ install: all
 	install drivers/libusbpre1.$(SHAREEXT) $(DESTDIR)/$(LIBDIR)/iguanaIR/libusbpre1.$(SHAREEXT)
 	install drivers/libusb.$(SHAREEXT) $(DESTDIR)/$(LIBDIR)/iguanaIR/libusb.$(SHAREEXT)
 	install -d $(DESTDIR)$(prefix)/include
-	install $(srcdir)/iguanaIR.h $(DESTDIR)$(prefix)/include/iguanaIR.h
+	install --mode 644 $(srcdir)/iguanaIR.h $(DESTDIR)$(prefix)/include/iguanaIR.h
 ifdef DARWIN
 	install -D --mode 644 osx/com.iguana.igdaemon.plist \
                               /Library/LaunchDaemons/com.iguana.igdaemon.plist
 	launchctl load -w /Library/LaunchDaemons/com.iguana.igdaemon.plist
 else
 	install -D $(srcdir)/iguanaIR.init $(DESTDIR)/etc/init.d/iguanaIR
-	install -D --mode 644 $(srcdir)/iguanaIR.options $(DESTDIR)/etc/default/iguanaIR
-	install -D --mode 644 $(srcdir)/udev/iguanaIR.rules \
-	                      $(DESTDIR)/lib/udev/rules.d/80-iguanaIR.rules
-	install --mode 755 $(SETOWNER) -d $(DESTDIR)/lib/udev/devices/iguanaIR
+	install -D --mode 644 $(srcdir)/iguanaIR.options $(DESTDIR)/etc/iguanaIR/iguanaIR
+#	install -D --mode 644 $(srcdir)/udev/iguanaIR.rules \
+#	                      $(DESTDIR)/lib/udev/rules.d/80-iguanaIR.rules
+#	install --mode 755 $(SETOWNER) -d $(DESTDIR)/lib/udev/devices/iguanaIR
 	install --mode 755 $(SETOWNER) -d $(DESTDIR)/dev/iguanaIR
 endif
 

