diff --git a/setup.py b/setup.py
index 1819fcf..89ccda9 100644
--- a/setup.py
+++ b/setup.py
@@ -3,18 +3,10 @@ from codecs import open
 from os import path
 from setuptools import setup, find_packages
 
-
-here = path.abspath(path.dirname(__file__))
-###################################################################
-# Get the long description from the README file
-with open(path.join(here, 'README.md'), encoding='utf-8') as f:
-    long_description = f.read()
-
 setup(
     name = APP_NAME,
     version = APP_VERSION,
     description = APP_DESCRIPTION,
-    long_description = long_description,
     url = APP_WEBSITE,
     author = APP_DEVELOPER,
     author_email = APP_DEVELOPER_EMAIL,
