diff -ruN pidgin-2.10.11-old/libpurple/protocols/jabber/message.c pidgin-2.10.11/libpurple/protocols/jabber/message.c
--- pidgin-2.10.11-old/libpurple/protocols/jabber/message.c	2016-02-01 19:26:27.881056000 -0500
+++ pidgin-2.10.11/libpurple/protocols/jabber/message.c	2016-02-01 21:18:38.292608710 -0500
@@ -156,6 +156,11 @@
 			jm->body = jabber_google_format_to_html(jm->body);
 			g_free(tmp);
 		}
+		
+		if(jm->nick) {
+			serv_got_alias(gc, jm->from, jm->nick);
+		}
+		
 		serv_got_im(gc, jm->from, jm->xhtml ? jm->xhtml : jm->body, 0, jm->sent);
 	}
 
@@ -770,6 +775,8 @@
 					&& purple_strequal(node, "http://jabber.org/protocol/commands")) {
 				jabber_adhoc_got_list(js, jm->from, child);
 			}
+		} else if (!strcmp(child->name, "nick") && !strcmp(xmlns, NS_NICK)) {
+			jm->nick = xmlnode_get_data(child);
 		}
 	}
 
diff -ruN pidgin-2.10.11-old/libpurple/protocols/jabber/message.h pidgin-2.10.11/libpurple/protocols/jabber/message.h
--- pidgin-2.10.11-old/libpurple/protocols/jabber/message.h	2016-02-01 19:26:27.899057000 -0500
+++ pidgin-2.10.11/libpurple/protocols/jabber/message.h	2016-02-01 19:44:47.541353715 -0500
@@ -52,6 +52,7 @@
 	char *password;
 	char *error;
 	char *thread_id;
+	char *nick;
 	enum {
 		JM_STATE_NONE,
 		JM_STATE_ACTIVE,
diff -ruN pidgin-2.10.11-old/libpurple/protocols/jabber/namespaces.h pidgin-2.10.11/libpurple/protocols/jabber/namespaces.h
--- pidgin-2.10.11-old/libpurple/protocols/jabber/namespaces.h	2016-02-01 19:26:27.907057000 -0500
+++ pidgin-2.10.11/libpurple/protocols/jabber/namespaces.h	2016-02-01 21:08:41.522363185 -0500
@@ -67,6 +67,9 @@
 /* XEP-0124 Bidirectional-streams Over Synchronous HTTP (BOSH) */
 #define NS_BOSH "http://jabber.org/protocol/httpbind"
 
+/* XEP-0172 User Nickname */
+#define NS_NICK "http://jabber.org/protocol/nick"
+
 /* XEP-0191 Simple Communications Blocking */
 #define NS_SIMPLE_BLOCKING "urn:xmpp:blocking"
 
